### DISCUSSION TITLE
6CF / GCF issue without changing wiring

### DISCUSSION THREAD

**Discussion by torei** [visualapproach/WiFi-remote-for-Bestway-Lay-Z-SPA#358] [General]
Did anyone had issues like that before?
I don't know what to do..

https://www.lay-z-spa.co.uk/support/gcf

I tried all of that except the electrician. The weird thing is that i havn't change the wiring.
What I did is changing the pathway for the cable. It's still the same and it's not broken.
Tried to use a different cable even this one is not working any more (and it was before a while ago..).
Then tried without the module, another electrical source but no luck at all.

I think it must be something inside the egg - or two cables were broken mysteriously.
Any idea? :I

what I did is the additional flex pipe (and a second cable):
![image](https://user-images.githubusercontent.com/6784876/183597640-34cdd4f5-78d1-4f76-855e-ed2aa37f0f55.png)

**Comment by torei** [comment]
I found out that my cables and house electric is working fine. Ground exists, all good.
So I guess, .. it must be the bestway hardware.. :(
Next step is to open that .. thing..

![image](https://user-images.githubusercontent.com/6784876/183840740-72eeffab-73ed-4dd9-bc27-bb4e379263b2.png)
(this picture was found from google..)

**Comment by torei** [comment]
found that one, nothing works..
https://youtu.be/mlJ289ds9_g

**Reply by visualapproach** [reply]
Sorry to hear. From what that guy said it appears to be common. Maybe reseat all wires in the pump and push the thermal reset button and checking at another house? I don't know. Hopefully someone can give advice.

**Reply by torei** [reply]
will wait for rain now.. on monday. maybe its really some strange "no earth on ground" issue because its that dry these days

**Reply by kingrun** [reply]
This will probably be some hardware issue or cables not seated properly but I do have to admit. I once plugged mine in to an extension cord without a ground pin and the GFC test went fine and it was working. But only happened once so go figure 🤷‍♂️

**Comment by torei** [comment]
so.. end of the story
something with the mainboard - i really dont have any clue what to do there

![image](https://user-images.githubusercontent.com/6784876/208871800-34f42128-68fd-4f73-b628-f2e4572324ce.png)

![image](https://user-images.githubusercontent.com/6784876/208923043-19bbace4-1559-4c09-9107-b8a98d9fdea5.png)

**Reply by JacekHA** [reply]
Do you know what was damaged on the motherboard?
